README.md

```diff
@@ -16,7 +16,7 @@ irb -r base64 -r ./lib/aes256gcm_decrypt.so
 token_data = Base64.decode64(File.read('test/token_data_base64.txt'))
 ciphertext = token_data[0..-17]
 tag = token_data[-16..-1]
-key = Base64.decode64(File.read('test/key_base64.txt'))
+key = File.read('test/key_hex.txt').scan(/../).map(&:hex).pack('c*')
```

Comment:
Or
```ruby
key = File.read('test/key_hex.txt').unpack('H*').first
```
You decide 🙂

Reply:
Btw. if someone reads this, I was wrong — it's `Array#pack` that should be used rather than `String#unpack`. @mt-clearhaus of course deciphered (pun intended) what I meant 🙂